src/g/zoomer.js

```diff
@@ -170,7 +171,9 @@ module.exports = Zoomer = Model.extend({
     }
 
     //@set "alignmentWidth", val
-    this.set("alignmentWidth", val)
+    // Note(ritik): Added extra pixels to fix the border getting clipped for last residue selection in sequences
```

Comment:
I don't get why need to increase the width of the zoomer section if the borders are part of the residue block.
Can you explain this?

Reply:
I'm not really sure regarding the exact cause but from the UI, some pixels seem missing in the width calculation. I also expected the borders to be completely visible without this adjustment since the borders are now a part of the residue block.

Reply:
Okay, I found the issue: the right border was not completely inside the residues even after my changes.
<img width="245" alt="Screenshot 2024-10-03 at 1 07 56 PM" src="https://github.com/user-attachments/assets/6c7e39d3-301e-45a5-a291-c09c2a406d25">
Let me fix it and remove these changes to the width.

Reply:
Done